tests/libvmi/vmi.go

```diff
@@ -117,10 +117,14 @@ func WithUefi(secureBoot bool) Option {
 }
 
 // WithSEV adds `launchSecurity` with `sev`.
-func WithSEV() Option {
+func WithSEV(isESEnabled bool) Option {
```

Comment:
What happens in the future when we'll also support SNP? Would it make sense to have a following function then? Or SNP has to be defined with ES?
```go
func WithSEV(isESEnabled, isSNPEnabled bool) Option {
```

Maybe it's better to define an explicit type to choose the different policies:
```go
type AmdSevPolicy string
const(
  Sev = "sev"
  SevES = "sev-es"
  SevSNP = "sev-snp"
)

func WithSEV(sevPolicy AmdSevPolicy) Option { ...
```

Reply:
I won't reason much about SNP since we don't know how libvirt will expose support for it. If it's exposed as a policy bit like SEV-ES, then I think the first approach is better since it will set the vmi spec directly. For SEV & SEV-ES, I think what is there currently is the cleanest approach.